### DISCUSSION TITLE
Translation use case?

### DISCUSSION THREAD

**Discussion by Tejaswgupta** [PaddlePaddle/PaddleOCR#12060] [General]
We've fine-tuned a transformers based translation engine which works on paragraph to ensure contextual translation. We had been using tesseract/HOCR for paragraph level extraction , but the HOCR library we used is obsolete now. PP-OCR seems a promising solution but I couldn't any resources on paragraph level extraction.

Can someone shed some light on this. Thanks!

**Comment by GreatV** [comment]
please refer: https://github.com/PaddlePaddle/PaddleOCR/blob/main/README_en.md#-tutorials

**Comment by Tejaswgupta** [comment]
@GreatV I've gone through the docs, the only relevant thing was PP-Structure but that's an overkill and would require more work to get components out of it for our use case.

**Comment by GreatV** [comment]
@Tejaswgupta  Sorry, as far as I know paddleocr doesn't do direct paragraph level detection and recognition.
